V3-Upgrade-Guide.md

```diff
@@ -80,6 +80,10 @@ The function `variadicArgumentToArray` is replaced by the function `transformVar
 
 The generic types of `JwtKeyMapping` is simplified so the second type argument `JwtKeysT` are always strings.
 
+#### Package `@sap-cloud-sdk/generator`
+
+The deprecated option `generateCSN` is removed. There is no replacement.
```

Comment:
[req] Please add some short description of how to do this instead. I highly doubt that this was ever used, but if it was, we should link to CAP and how it can be done (it's in the ticket).

Reply:
I added a link to the CDS CAP docu